dioptas/controller/MaskController.py

```diff
@@ -33,6 +33,11 @@
 
 
 class MaskController(object):
+
+    DEFAULT_MASK_FILTER = 'Mask (*.mask)'
```

Comment:
I hesitate to add `.npy` and `.edf` also to the default:
```suggestion
    DEFAULT_MASK_FILTER = 'Mask (*.mask *.npy *.edf)'
```
The remaining of the code should work fine.